docs/meetings/backend/Meeting 041.md

```diff
@@ -0,0 +1,10 @@
+# Backend team meeting #41
+
+When: 15/2/22, 21:30 UTC.
+Attending: Aapeli, Farley
+
+## Notes
+
+* Farley migrated weblate, wiki, otobo to k8s! Hooray!
+* Discussion on Kubernetes, Helm, etc
+* Next steps: moving CI/CD to GitHub, then deploying next onto k8s
```

Comment:
> moving CI/CD to GitHub

Out of curiousity, why?

Reply:
So it's all in the same place

Reply:
With GitHub Actions?

Reply:
Yep

Reply:
Yeah if you host your own runners

Reply:
+ some stuff for public repos

Reply:
@AndrewFarley

Reply:
@lucaslcode Chiefly because of complexity (simplicity) and integration.  Gitlab is wonderful, I'm a huge fan and their CI is one of (if not) the best in the world as far as I'm concerned, I'm a solutions partner for Gitlab personally.  But our code lies in Github with no plan to move it, and the integration of CI/CD between the two systems leaves a lot to be desired.  Github will only show "one" pipeline which in Gitlab will often be many jobs.  This leads to a poor experience, and adds complexity when it comes to automation and integration (such as Slack).  We've decided to simplify and unify our technology stack moving forward.  Kubernetes is one of those pivots, and centralizing on Github Actions is another.  I'm taking ownership of pivoting all our existing automations over a little at a time.

When this process is completed we'll be able to use the full feature-set of Github, including features like Environments to be able to check what is deployed where, perform rollouts/rollbacks easily at the click of a button, and have an full view into the status of CI without requiring any supporters to have logins and understand how to use both Gitlab and Github.